app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<div>
+	<h2>All Users</h2>
+	<ul>
+		<% @users.each do |user| %>
+			<li>
+				<%= user.email %>
+				<% if user.role == "admin" %>
```

Comment:
Helpers for these types of checks would make the code a bit cleaner. Something like `user.admin?`